packages/path_provider/example/integration_test/path_provider_test.dart

```diff
@@ -2,10 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart=2.9
-
```

Comment:
Hmm. As of Flutter 2.2.1, integration_test has not been migrated to null safety yet. You are supposed to do this migration after the next Flutter stable release, although the code will just run okay with null-safety and only warnings will be generated in the IDE.

Reply:
I've switched to tag 2.2.1 in flutter branch and looked into the integration_test package, is it perhaps not the right way to check?

May also apply to #158?

Reply:
Ah, you're right. It's already been migrated. I was thinking of Flutter 2.0.1 but not 2.2.1.

https://github.com/flutter/flutter/blob/2.0.1/packages/integration_test/pubspec.yaml
https://github.com/flutter/flutter/blob/2.2.1/packages/integration_test/pubspec.yaml